src/components/shows_container/ShowsContainer.js

```diff
@@ -9,16 +9,16 @@ const ShowsContainer = ({ year }) => {
   const [isLoading, setIsLoading] = useState(true);
 
   const memo = useCallback(() => {
-    const renderShows = async () => {
+    const fetchShows = async () => {
       await getShows(year);
       setIsLoading(false)
     }
-    renderShows()
-  }, [year, getShows])
+    fetchShows()
+  }, [year])
```

Comment:
All good! I wasn't getting an infinite loop when we created this callback function, but maybe this ask @justincanthony mentor about.